doc/Auxiliary-Files.md

```diff
@@ -4,5 +4,5 @@
     * 制御則の解説
     * パラメータファイルの設定方法概要
 
-* [libypspur コマンド一覧](https://github.com/openspur/yp-spur/raw/master/doc/files/libypspur_cheatsheet.pdf)
+* [libypspur コマンド一覧](files/libypspur_cheatsheet.pdf)
```

Comment:
This would be better to be raw.githubusercontent.com URL.
https://raw.githubusercontent.com/openspur/yp-spur/master/doc/files/libypspur_cheatsheet.pdf

Reply:
Addressed.